File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/summer_outfit.py
degrees = int(input())
day_part = str(input())
outfit = ""
shoes = ""

if 10 <= degrees <= 18:
    if day_part == "Morning":
        outfit = "Sweatshirt"
        shoes = "Sneakers"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Afternoon":
        outfit = "Shirt"
        shoes = "Moccasins"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Evening":
        outfit = "Shirt"
        shoes = "Moccasins"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")

elif 18 < degrees <= 24:
    if day_part == "Morning":
        outfit = "Shirt"
        shoes = "Moccasins"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Afternoon":
        outfit = "T-Shirt"
        shoes = "Sandals"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Evening":
        outfit = "Shirt"
        shoes = "Moccasins"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")

elif degrees >= 25:
    if day_part == "Morning":
        outfit = "T-Shirt"
        shoes = "Sandals"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Afternoon":
        outfit = "Swim Suit"
        shoes = "Barefoot"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")
    elif day_part == "Evening":
        outfit = "Shirt"
        shoes = "Moccasins"
        print(f"It's {degrees} degrees, get your {outfit} and {shoes}.")

File: JS/JavaScript/JS-Basics/Conditional StatementsAdvancedExercise/SummerOutfit.js
function summerOutfit(inputs) {

    let degrees = Number(inputs[0])
    let dayPart = String(inputs[1])

    outfit = ""
    shoes = ""

    if (degrees >= 10 && degrees <= 18) {
        if (dayPart == 'Morning'){
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        }
        else if (dayPart == "Afternoon") {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
        else if (dayPart == 'Evening') {
            outfit = "Shirt";
            shoes = "Moccasins";
        };

    }

    else if (degrees > 18 && degrees <= 24) {
        if (dayPart == 'Morning'){
            outfit = "Shirt";
            shoes = "Moccasins";
        }
        else if (dayPart == "Afternoon") {
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
        else if (dayPart == 'Evening') {
            outfit = "Shirt";
            shoes = "Moccasins";
        };
    }

    else if (degrees >= 25) {
        if (dayPart == 'Morning'){
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
        else if (dayPart == "Afternoon") {
            outfit = "Swim Suit";
            shoes = "Barefoot"; 
        }
        else if (dayPart == 'Evening') {
            outfit = "Shirt"; 
            shoes = "Moccasins"; 
        };
    };


    console.log(`It's ${degrees} degrees, get your ${outfit} and ${shoes}.`)

}


summerOutfit(["16",
"Morning"])
